### ISSUE TITLE
[Bug]: error:can't find column in context's map map[4294967296:[0 0] 4294967297:[0 1] 4294967298:[0 2] 4294967299:[0 3] 4294967300:[0 4] 42949672960:[1 0]]

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#3311]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):0.5.0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

[ERROR]
[SCRIPT   FILE]: cases/subquery/subquery-with-exists.sql
[ROW    NUMBER]: 88
[SQL STATEMENT]: DELETE FROM t1 WHERE topic IN (SELECT DISTINCT topic FROM t2 WHERE NOT EXISTS(SELECT * FROM t3 WHERE numeropost=topic));
[EXPECT RESULT]:

[ACTUAL RESULT]:
can't find column in context's map map[4294967296:[0 0] 4294967297:[0 1] 4294967298:[0 2] 4294967299:[0 3] 4294967300:[0 4] 42949672960:[1 0]]

[ERROR]
[SCRIPT   FILE]: cases/subquery/subquery-with-exists.sql
[ROW    NUMBER]: 89
[SQL STATEMENT]: select * from t1;
[EXPECT RESULT]:
mot     topic   dt      pseudo
joce    1       null    joce
[ACTUAL RESULT]:
mot     topic   dt      pseudo
joce    1       null    joce
test    2       null    test

[ERROR]
[SCRIPT   FILE]: cases/subquery/subquery-with-exists.sql
[ROW    NUMBER]: 124
[SQL STATEMENT]: SELECT table1.f1, table2.f1_key FROM t1 AS table1, t2 AS table2
WHERE EXISTS
(
SELECT DISTINCT f1_key
FROM t2
WHERE f1_key != table2.f1_key AND f1_key >= table1.f1 );
[EXPECT RESULT]:
f1      f1_key
s       j
v       j
s       v
v       v
s       c
v       c
s       m
v       m
s       d
v       d
s       d
v       d
s       y
v       y
s       t
v       t
s       d
v       d
s       s
v       s
[ACTUAL RESULT]:
can't find column in context's map map[4294967296:[0 0] 8589934592:[0 1] 30064771072:[1 0]]

**Comment by fengttt** [comment]
Fix error message and move it to 0.6.

**Comment by aressu1985** [comment]
fixed


mysql> SELECT DISTINCT topic FROM t2 WHERE NOT EXISTS(SELECT * FROM t3 WHERE numeropost=topic);
+-------+
| topic |
+-------+
|     2 |
+-------+
1 row in set (0.00 sec)

mysql> 
mysql> 
mysql> DELETE FROM t1 WHERE topic IN (SELECT DISTINCT topic FROM t2 WHERE NOT EXISTS(SELECT * FROM t3 WHERE numeropost=topic));
Query OK, 1 row affected (0.00 sec)

mysql> select * from t1;
+------+-------+------+--------+
| mot  | topic | dt   | pseudo |
+------+-------+------+--------+
| joce |     1 | NULL | joce   |
+------+-------+------+--------+
1 row in set (0.00 sec)


mysql> -- @bvt:issue#3311
mysql> SELECT table1.f1, table2.f1_key FROM t1 AS table1, t2 AS table2
    -> WHERE EXISTS
    -> (
    -> SELECT DISTINCT f1_key
    -> FROM t2
    -> WHERE f1_key != table2.f1_key AND f1_key >= table1.f1 );
+------+--------+
| f1   | f1_key |
+------+--------+
| v    | j      |
| v    | v      |
| v    | c      |
| v    | m      |
| v    | d      |
| v    | d      |
| v    | y      |
| v    | t      |
| v    | d      |
| v    | s      |
| s    | j      |
| s    | v      |
| s    | c      |
| s    | m      |
| s    | d      |
| s    | d      |
| s    | y      |
| s    | t      |
| s    | d      |
| s    | s      |
+------+--------+
20 rows in set (0.00 sec)
